superagi/vector_store/chromadb.py

```diff
@@ -0,0 +1,70 @@
+
+
+import uuid
+from typing import Any, Callable, Optional, Iterable, List
+
+from superagi.vector_store.document import Document
+from superagi.vector_store.base import VectorStore
+from superagi.vector_store.embedding.openai import BaseEmbedding
+from superagi.chromadb.client import ChromaDBClient
+
+
+class ChromaDB(VectorStore):
+    def __init__(
+        self,
+        client: ChromaDBClient,
```

Comment:
Can you add ChromaDBClient also in the PR?

Reply:
I have added. Kindly have a look.